pom.xml

```diff
@@ -562,15 +560,41 @@
                     </executions>
                 </plugin>
                 <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
```

Comment:
You can omit the groupId here. Not sure it's done for the existing ones but we try to do it for the new ones.